.gitignore

```diff
@@ -6,3 +6,7 @@ node_modules/
 
 # Build output
 dist/
+
+# Editors
+.idea
+.vscode
```

Comment:
Can you please use trailing slashes to indicate those are direcories? Otherwise the files `.idea` and `.vscode` are ignored as well, which does no harm but with / it is easier to read